apps/doji-backend/src/session/session.controller.ts

```diff
@@ -56,4 +67,25 @@ export class SessionController {
     await this.sessionService.deleteSessionParticipant(body.sessionId, user)
     return 'OK'
   }
+
+  @Get('/session/:id')
+  @ApiOperation({ description: 'Retrieve session info' })
+  async getSession(@Param('id', ParseIntPipe) id: number): Promise<ISessionInformationDTO> {
+    const session = await this.sessionService.getSessionInfo(id)
+    if (!session) {
+      throw new NotFoundException('No such session id')
+    }
+    //TODO Populate Value
+    return {
+      id: session.id,
+      reviews: session.reviews.getItems().map((review) => ({
+        id: review.id,
+        rating: review.rating,
+        content: review.content,
+        authorName: `${review.user.firstName} ${review.user.lastName}`,
+        createdAt: review.createdAt,
+      })),
+      reviewStat: await this.sessionService.calculateReviewStatForSession(session),
```

Comment:
why not just `session.reviews.getItems()` na?

Reply:
Not all properties should be output to the user.